tests/bootstrap.php

```diff
@@ -8,14 +11,9 @@
 
 $loader->add('Jackalope', __DIR__.'/../vendor/jackalope/jackalope/tests');
 
-/** make sure we get ALL infos from php */
-error_reporting(E_ALL | E_STRICT);
-
-### Load classes needed for jackalope unit tests ###
-require 'Jackalope/Transport/Jackrabbit/JackrabbitTestCase.php';
-
 ### Load the implementation loader class ###
-require 'inc/ImplementationLoader.php';
+require_once __DIR__ . '/ImplementationLoader.php';
+require_once __DIR__ . '/JackrabbitFixtureLoader.php';
```

Comment:
right. should have looked at the file titles in the diff as well to see that you move them.